app/controllers/admin/creator_settings_controller.rb

```diff
@@ -1,15 +1,26 @@
 module Admin
   class CreatorSettingsController < Admin::ApplicationController
-    ALLOWED_PARAMS = %i[community_name logo_svg primary_brand_color_hex invite_only_mode public checked_code_of_conduct
-                        checked_terms_and_conditions].freeze
+    ALLOWED_PARAMS = %i[community_name primary_brand_color_hex invite_only_mode public checked_code_of_conduct
+                        checked_terms_and_conditions logo].freeze
```

Comment:
Super nitpicky and not directly related to this PR but we often keep these constants alphabetically sorted and since we're already modifying it maybe this would be a great time to make this minor change?

```suggestion
    ALLOWED_PARAMS = %i[checked_code_of_conduct checked_terms_and_conditions community_name  
                        invite_only_mode logo primary_brand_color_hex public]
```

Reply:
✅